src/partr.c

```diff
@@ -298,6 +321,7 @@ JL_DLLEXPORT int jl_enqueue_task(jl_task_t *task)
 //  sleep_check_after_threshold() -- if sleep_threshold ns have passed, return 1
 static int sleep_check_after_threshold(uint64_t *start_cycles)
 {
+    JULIA_DEBUG_SLEEPWAKE( return 1; ) // hammer on the sleep/wake logic much harder
```

Comment:
Could you add semicolons after these? This seems to confuse emacs auto-indent.

Reply:
Is that better?